### ISSUE TITLE
Upgrade to Docusaurus v3

### ISSUE THREAD

**Issue by ElliotFriend** [stellar/stellar-docs#355]
Docusaurus v3 is out now, and (pending a plugin update) we should be using it.

More context and thoughts:

- the particular plugin we've been waiting on is [docusaurus-openapi-docs](https://github.com/PaloAltoNetworks/docusaurus-openapi-docs), which is working on a [v3 upgrade](https://github.com/PaloAltoNetworks/docusaurus-openapi-docs/pull/660). it's possible using one of the `v3.0.0-beta.x` releases will be suitable
- We should also take this time to make sure all of our tooling, gh actions, `package.json`, `.nvmrc`, whatever else are using a consistent version of nodejs (`v20` seems ideal)

**Comment by waldmatias** [comment]
When this happens... could we **please** enable "es" support for i18n? We would be more than happy to provide es-versions of the entire docs in preparation for greater LATAM awareness of Stellar.

**Comment by ElliotFriend** [comment]
Hi, @waldmatias!! I started typing a "brief" comment that got a lot longer than I expected hehe

**TL;DR:** We're actively considering how to support "es" translation for our technical docs.

That's an excellent request! We're actually in the process of evaluating exactly that option! They're currently working on providing Spanish-language i18n for the main www.stellar.org site. The Spanish translations aren't _yet_ available, but I'm told they should be viewable in the next couple weeks or so? (don't quote me on that timeline precisely lol).

I don't know the specifics of how they're translating everything, but it has something to do with automating it with ChatGPT. The process has been proofed/checked/refined by folks who natively speak Spanish, though not with a technical background. Once those pages are live and available, I'd be really curious to hear your impression of it (especially since you're really familiar with Stellar). I can't speak Spanish any further than counting to 10 and asking where the restroom is, so I can't really speak to its quality 🤷🏻‍♂️

**Comment by waldmatias** [comment]
Thanks @ElliotFriend, good to see work is already under way. Though I have to say, it has been my experience that translations for technical-oriented (specific) materials still yield the best results (and less rework) when tackled manually. I would be happy to proofread some of the material once there's at least a draft. 
In any case, concerning implementation for v3 Docusaurus, you need no go further than checking out pnpm's docs: 
https://github.com/pnpm/pnpm.io/blob/main/docusaurus.config.js#L308 to get some ideas.

**Comment by ElliotFriend** [comment]
Yeah, I'm interested to see how feasible the automated system is, too.

I'm **SO GLAD** you shared that link to pnpm's docs repo. I didn't know they used docusaurus, and it looks like there are some really useful techniques we might be able to use along the way (even non-translation stuff). Thank you!!

**Comment by ElliotFriend** [comment]
With #405 merged in, we're ready to proceed onward and upward to docusaurus v3! 🎉
